tests/general/LineCountBearTest.py

```diff
@@ -40,3 +40,35 @@ def test_exclude_blank_lines(self):
                                 severity=RESULT_SEVERITY.NORMAL,
                                 file='default')],
             filename='default')
+
+    def test_min_lines_per_file_1(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('min_lines_per_file', 5))
+        self.check_results(
+            self.uut, ['line 1', 'line 2', 'line 3', 'line 4'],
+            [Result.from_values('LineCountBear',
+                                'This file had 4 lines, which is 1 lines less '
+                                'than the minimum limit specified.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+
+    def test_min_lines_per_file_2(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('min_lines_per_file', 1))
+        self.check_results(
+            self.uut, [],
+            [Result.from_values('LineCountBear',
+                                'This file had 0 lines, which is 1 lines less '
+                                'than the minimum limit specified.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+
+    def test_valid_settings(self):
```

Comment:
Since we have `exclude_blank_lines` available now, it would be good to integrate all the valid settings in this test. :)